packages/tools/tests/test/audioV2/utils/audioEngineV2.utils.ts

```diff
@@ -0,0 +1,158 @@
+import type { Nullable } from "@dev/core/types";
+import { test } from "@playwright/test";
+import { getGlobalConfig } from "@tools/test-tools";
+
+/**
+ * The maximum pulse volume in the sound test file containing the pulse train.
+ */
+const MaxPulseVolume = 0.1;
+
+// Declarations for babylonServer/public/audiov2-test.js
+declare global {
+    let audioTestConfig: AudioTestConfig;
+    let audioTestResult: AudioTestResult;
+
+    class AudioV2Test {
+        public static AfterEachAsync(): Promise<void>;
+        public static BeforeEachAsync(): Promise<void>;
+        public static CreateAudioEngineAsync(options?: Partial<BABYLON.IWebAudioEngineOptions>): Promise<BABYLON.AudioEngineV2>;
+        public static CreateSoundAsync(source: string | string[], options?: Partial<BABYLON.IStaticSoundOptions>): Promise<any>;
+        public static GetResultAsync(): Promise<AudioTestResult>;
+        public static WaitAsync(seconds: number): Promise<void>;
+    }
+}
```

Comment:
Maybe add a .d.ts file for this?

Reply:
not sure it is needed, unless you need it as a reference outside of this test